client/src/util/CommandUtil.ts

```diff
@@ -13,12 +13,16 @@
 */
 'use strict';
 
+import * as vscode from 'vscode';
 import * as child_process from 'child_process';
 import stripAnsi = require('strip-ansi');
 
 import * as childProcessPromise from 'child-process-promise';
 import { ConsoleOutputAdapter } from '../logging/ConsoleOutputAdapter';
 import { OutputAdapter } from '../logging/OutputAdapter';
+import * as fs from 'fs-extra';
+import * as homeDir from 'home-dir';
+import { PackageTreeItem } from '../explorer/model/PackageTreeItem';
```

Comment:
This isn't used, if it is and i'm blind you shouldn't be creating packageTreeItems in here it should do this in the packageProvider